js/script.js

```diff
@@ -258,12 +258,11 @@ function savebtn() {
         sendToMyJSON(jsonstring);
     }
 
-}
-
-window.closebtn = function () {
-    document.getElementById("blueprint").style.display = "none";
 };
 
+function closebtn() {

```

Comment:
I think this is used from the html. If it is it should be added to the window obiect.